### ISSUE TITLE
Windows 10 and smb

### ISSUE THREAD

**Issue by KarterJK** [mike42/escpos-php#179]
First, thank you for the library, saved my arse....

I am trying to connect to a shared Windows printer using

Windows 10
XAMPP server
PHP Version 5.6.8

I can do the  'copy testfile "computer/ESPSON Receipt" and it works fine
but when trying to use the smb:// method it fails

I have a windows 7 machine that it works with but not Windows 10
I am thinking it may be related to the new smb3 version it uses

any answers?  suggestions?

**Comment by mike42** [comment]
What sort of error do you get when you try to print via a `new WindowsPrintConnector("smb://computer/ESPSON Receipt")`?

I'm not a Windows user myself, but searching the bug tracker for "XAMPP" or "Windows permissions" turns up some threads where people may have had the same issue.

**Comment by KarterJK** [comment]
Thank you for the response..

I can't recall exactly.  The software & printer were originally setup as a local connection, but new equipment came in and the printer was  moved.  Since, I could not resolve it, and this was needed immediately,  I just attached it locally via a 50' foot usb extension cable.  So, I would have to reset it all up again, to test it.

I tried setting permissions on every printer file I could find, but didn't seem to help.  If I get the chance I will test it again, as share setup and inform you of the results.  Just may not be any time soon.

Thank you again.
